<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Train</class>
 <widget class="QDialog" name="Train">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="numEpochs">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Epochs</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="earlyStopping">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Early Stopping</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="threshold">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveEpoch">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Epoch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>571</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="trainBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>earlyStopping</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveEpoch</sender>
   <signal>toggled(bool)</signal>
   <receiver>filename</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
